presto-tests/pom.xml

```diff
@@ -216,6 +216,7 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
                 <configuration>
+                    <argLine>-Xms4g -Xmx4g</argLine>
```

Comment:
I think this needs to be included in the correct profile. I see two profiles in the same file. `ci` and `ci-only` . Do we know which is the correct profile.?

Reply:
This should apply to both right if we specify here right?

Reply:
👍  LGTM